test/topology_experimental_raft/test_tablets.py

```diff
@@ -440,7 +455,7 @@ async def check_with_down(down_node):
         for r in rows:
             assert r.c == r.pk
 
-    await manager.rolling_restart(servers, with_down=check_with_down)
+    cql = await safe_rolling_restart(manager, servers, with_down=insert_with_down)
```

Comment:
This dropped the use of check_with_down

@raphaelsc

Reply:
oops, will send a fix.